README.md

```diff
@@ -150,6 +150,10 @@ properties:
 
 ## Dependency management
 To add a new dependency to the project, add the package information to glide.yaml and execute `glide up -v`
+* NOTE: some versions of glide might crash or generate incomplete vendor/ folder when ran against this repo. In this case you may wanna try getting the latest version:
```

Comment:
s/generate/generate an/
s/wanna/want to/

Reply:
thanks